src/tasks/tasks.py

```diff
@@ -1457,12 +1339,15 @@ def load_data(self, path, max_seq_len):
         log.info("\tFinished loading JOCI data.")
 
 
+@register_task('wmt_debug', rel_path='wmt_debug/', max_targ_v_size=5000)
```

Comment:
do we want to keep this debug example?

Reply:
Let's leave as a TODO for now so that this PR stays a no-op.